tests/set_test.go

```diff
@@ -127,3 +127,18 @@ func TestSetWithOptions(t *testing.T) {
 		})
 	}
 }
+
+func TestSetWithExat(t *testing.T) {
+	conn := getLocalConnection()
+	defer conn.Close()
+	Etime := strconv.FormatInt(time.Now().Unix()+10, 10)
+
+	assert.Equal(t, "OK", fireCommand(conn, "SET k v EXAT "+Etime), "Value mismatch for cmd SET k v EXAT "+Etime)
```

Comment:
Add test for the negative condition where user can provide `EXAT` and `PXAT` in the same command.

Reply:
Hi @soumya-codes, I have updated the code base as per your comments. Please review.